src/olympia/blocklist/admin.py

```diff
@@ -523,20 +550,15 @@ def has_change_permission(self, request, obj=None):
         else:
             return super().has_change_permission(request, obj=obj)
 
+    def has_delete_permission(self, request, obj=None):
+        # TODO: restore this functionality so bulk delete isn't the only option
```

Comment:
We should file a follow-up issue and link it here.

Reply:
https://github.com/mozilla/addons-server/issues/13555 - its in the project